docs/rackspace-elastic-engineering-and-optimizer/index.rst

```diff
@@ -5,14 +5,14 @@ Rackspace Elastic Engineering and Optimizer +
 =============================================
 
 Rackspace knows that our customers have varying needs at different stages of
-their cloud journey. That’s why we provide a set of support offers that solve
-for your needs at any stage of your cloud lifecycle. Our services provide you
-access to certified cloud specialists through a flexible POD-based model,
-experts who can implement industry best cloud monitoring technologies, cost
-optimization recommendations, experts to solve your IT challenges, and other
+their cloud journey. That’s why we provide a set of support offers that address
+your needs at any stage of your cloud lifecycle. Our services provide you
+access to certified cloud specialists through a flexible POD-based model. We have
+experts who can implement industry-best cloud monitoring technologies, suggest cost
+optimization recommendations, solve your IT challenges, and offer other
 value-added services, all backed by AWS certified engineers and architects.
```

Comment:
Should final clause be separated? Also, we should probably review for anything that mentions AWS in these  GCP docs 😄 

```suggestion
value-added services — all backed by GCP-certified engineers and architects.
```